devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -156,8 +156,9 @@ def configureHealthCheck() {
 		runEvery5Minutes("healthPoll")
 		// Device-Watch allows 2 check-in misses from device
 		sendEvent(name: "checkInterval", value: hcIntervalMinutes * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+		def childEvent = createEvent([name: "checkInterval", value: hcIntervalMinutes * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID]])
```

Comment:
Could create the eventMap once to decrease the potential for them getting set differently